core/src/main/scala/org/http4s/ProcessApp.scala

```diff
@@ -0,0 +1,39 @@
+package org.http4s
+
+import org.http4s.internal.compatibility._
+
+import scalaz.{-\/, \/-}
+import scalaz.concurrent.Task
+import scalaz.stream.Process
+import scalaz.stream.Process.eval_
+import scalaz.stream.{async, wye}
+
+trait ProcessApp {
```

Comment:
I'm confused about `org.http4s.ProcessApp` versus `org.http4s.util.ProcessApp`.  If I `git rm core/src/main/scala/org/http4s/ProcessApp.scala`, `sbt test` still passes.  If there's a reason we need the former, a comment might be worthwhile.